source/server/README.md

```diff
@@ -34,7 +34,7 @@ static_resources:
           port_value: 10000
       filter_chains:
         - filters:
-            - name: envoy.http_connection_manager
+            - name: envoy.filters.network.http_connection_manager
```

Comment:
Was this automatically generated or does it need a merge? Not sure if we intended to have this change in this PR.

Reply:
I merged master into this, it seems that `source/server/README.md` needs an update from v2 to v3. I can do that here or we can move that into a separate PR, please let me know your preference.

Reply:
It is fine to leave it here, I just wasn't sure it was intended since the PR description doesn't mention it. I will update it and merge. Thank you.